packages/codemod/src/test/__testfixtures__/update-telemetry-data.output.ts

```diff
@@ -0,0 +1,37 @@
+// @ts-nocheck
+import { generateText, streamText, generateObject } from 'ai';
+import { createOpenAI } from '@ai-sdk/openai';
+
+const provider = createOpenAI();
+
+// Reading telemetry data
+async function handleResponse() {
+  const result = await generateText({
+    model: provider('gpt-4'),
+    prompt: 'Hello'
+  });
+
+  // Direct property access
+  console.log('Finish reason:', result.telemetry?.attributes['ai.response.finishReason']);
+  console.log('Generated text:', result.telemetry?.attributes['ai.response.text']);
+  console.log('Tool calls:', result.telemetry?.attributes['ai.response.toolCalls']);
+
+  // Object destructuring
+  const {
+    'ai.response.msToFirstChunk': latency,
+    'ai.response.object': resultObj
+  } = result.telemetry?.attributes ?? {};
+
+  if (latency > 1000) {
+    console.warn('High latency detected');
+  }
+
+  // Accessing all paths
+  const metrics = {
+    finish: result.telemetry?.attributes['ai.response.finishReason'],
+    text: result.telemetry?.attributes['ai.response.text'],
+    tools: result.telemetry?.attributes['ai.response.toolCalls'],
+    latency: result.telemetry?.attributes['ai.response.msToFirstChunk'],
+    object: result.telemetry?.attributes['ai.response.object']
```

Comment:
it is not possible to access telemetry like this. it goes to oauth. this might not be something to address in a codemod at all, it only affects a few telemetry providers like langchain that need to update their integrations and was listed for completeness only

Reply:
(that being said, it would be useful to expose telemetry like this in the future - but it's not something we do today)

Reply:
ok, great, thanks -- you proactively answered the note I just put in Slack!